test/WebSites/ModelBindingWebSite/Models/Product.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace ModelBindingWebSite
```

Comment:
`ModelBindingWebSite.Models` for this and other classes that belong (and remain) in this directory